src/renderer/components/ImageCropper/index.tsx

```diff
@@ -103,11 +114,71 @@ export default function ImageCropper({
     })()
   }
 
+  const onZoomIn = () => {
+    zoom.current += 0.01
+    moveImages(posX.current, posY.current)
+  }
+
+  const onZoomOut = () => {
+    zoom.current -= 0.01
+    moveImages(posX.current, posY.current)
+  }
+
   const onZoomReset = () => {
+    if (!cutImage.current || !fullImage.current) {
+      return
+    }
     posX.current = 0
     posY.current = 0
     zoom.current = initialZoom.current
     moveImages(0, 0)
+    if (rotation.current > 0) {
+      cutImage.current.src = filepath
+      fullImage.current.src = filepath
+    }
```

Comment:
IDK if `onZoomReset` is also supposed to reset rotation (then it needs a name change), but it does. But `rotation.current` here remains unchanged, so it gets out of sync with the actual rotation of the image, so sometimes the "reset" button doesn't actually reset rotations, and sometimes it does.

Reply:
Thanks, good catch!